tensorflow_lite_support/c/task/processor/classification_result_c_api.cc

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
It should be "2021". Same for the other files.

Reply:
Done